### PR TITLE
New package: CABS.SalesPrix version 6.15.0.0

### PR DESCRIPTION
**Pull request by KlausGue**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113765&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/c/CABS/SalesPrix/6.15.0.0/CABS.SalesPrix.locale.de-DE.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: CABS.SalesPrix
+PackageVersion: 6.15.0.0
+PackageLocale: de-DE
+Publisher: CAB+S GmbH
+PackageName: SalesPrix
```
Comment (stephengillie) [inline manifests/c/CABS/SalesPrix/6.15.0.0/CABS.SalesPrix.locale.de-DE.yaml:8]:
```suggestion
PackageName: SalesPrix
PackageUrl: https://www.scanprix.de
```

**Comment by KlausGue** [comment]
> /AzurePipelines run

my domain in www.scanprix.de

**Comment by ItzLevvie** [comment]
[Policy] Needs-Author-Feedback

Please commit suggestions from https://github.com/microsoft/winget-pkgs/pull/113765#discussion_r1279882786

**Pull request closed without merging**